### PR TITLE
Update fluent-plugin-google-cloud gem version to 0.7.1.

### PR DESCRIPTION
**Pull request by qingling128**

### CODE REVIEW

File: plugin_gems.rb [hunk 1]
```diff
@@ -18,7 +18,7 @@
 download "string-scrub", "0.0.5"
 download "fluent-plugin-rewrite-tag-filter", "1.5.5"
 download "fluent-plugin-td-monitoring", "0.2.2"
-download "fluent-plugin-google-cloud", "0.7.0"
+download "fluent-plugin-google-cloud", "0.7.1"
```
Comment (igorpeshansky) [inline plugin_gems.rb]: This hasn't been released yet, has it?

Comment (qingling128) [reply plugin_gems.rb]: Right

**Comment by qingling128** [comment]
No longer needed.

**Comment by qingling128** [comment]
Reopening this as this is relevant now.

**Pull request merged**